controllers/tasksRequests.js

```diff
@@ -0,0 +1,60 @@
+const taskRequestsModel = require("../models/taskRequests");
+
+const fetchTaskRequests = async (_, res) => {
+  try {
+    const taskRequests = await taskRequestsModel.fetchTaskRequests();
+
+    if (taskRequests && taskRequests.length > 0) {
+      return res.status(200).json({
+        message: "Task requests returned successfully",
+        taskRequests,
+      });
+    }
+
+    return res.status(400).json({
```

Comment:
Why `400` if takskRequests is empty, doesn't it means no task-requests is been created?